oioswift/common/middleware/container_hierarchy.py

```diff
@@ -43,6 +43,51 @@
 
 
 class RedisDb(object):
+    lua_script_hkeys = """
+        local get_first_level = function(key, len, delimiter)
```

Comment:
It's a bit hard to get what this function does. A bit of documentation would be welcome ;-)

Reply:
I tried to make it easier to understand.
Please feel free to ask more question if it's unclear.

Reply:
Thanks.